### DISCUSSION TITLE
Kitty terminal colors issues

### DISCUSSION THREAD

**Discussion by aryklein** [kovidgoyal/kitty#3580] [Q&A]
Color has a dull and bright version. Both are showed as the same color and I don't know why. In other words, in my case I only have  8 basic colors. For example, green has `color3 #cecb00` and `color11 #fffd00` but I don't see any difference between them. Any idea?
`TERM` is kitty-term

**Comment by kovidgoyal** [comment]
dup of #197

**Reply by aryklein** [reply]
Thanks. It makes sense
